event_survey/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module extends the functionality of event and survey module. It allows
+users create new surveys in event, form survey template, and keep track linked
```

Comment:
```suggestion
users to mark an existing survey as a survey template, so it can be reused to generate new surveys on events according to the template, without the need to create the survey from scratch.```